webui.py

```diff
@@ -2030,9 +2036,9 @@ def run_RealESRGAN(image, model_name: str):
 
 img2img_defaults = {
     'prompt': '',
-    'ddim_steps': 50,
+    'ddim_steps': 25,
     'toggles': [1, 4, 5],
-    'sampler_name': 'k_lms',
+    'sampler_name': 'k_euler_a',

```

Comment:
Same as above, do not add undocumented changes.

Reply:
Good point. These are defaults of my own convenience and I have yet to figure out a way of breaking up my PR and squashing down my commit tree.

Reply:
I recommend using a config file to change the defaults.
That way your change in parameters is completely decoupled from any changes to code.

Alternatively use git to stash your private changes, do any dev work that you want to do, commit your changes, and apply the stashed private changes.

Reply:
Thanks for the tip.